### ISSUE TITLE
Audio and WAV view not synced.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [quemoni/subtitleedit#34]
```
What steps will reproduce the problem?
1.Create wav file in Subtitle Edit, click on area; wav successfully created.
2.Start to add subtitles
3.click the time bar in the wav window back a few secs to adjust/verify and the 
sound coming out of the speakers and visual wave are not synced!
4 if I hit the Stop button in the Video section and play from the beginning, 
the issue goes away. This gets to be a pain in the butt if you have a very long 
video.

What is the expected output? What do you see instead?
The audio coming out of the speakers should line-up with the visual wave on the 
screen.  It only gets out of sync if I click back on the time line and then 
play.

What version of the product are you using? On what operating system?
3.2 31504
Windows 7, 32-Bit

Please provide any additional information below.
This has happened before, but not all the time.  This video file was a .vob 
from a DVD.  I renamed the file to .mpg.  I've done this before with no issues.
```

Original issue reported on code.google.com by `ozri...@gmail.com` on 17 Jul 2011 at 12:06

**Comment by GoogleCodeExporter** [comment]
```
Having the same problem on Windows Vista, 64-bit.

Not only when clicking back on the time line, but also when selecting a 
subtitle in the list view and going to its position (by double-clicking it or 
using the button "Go to subposition and pause").
```

Original comment by `matthias...@gmail.com` on 21 Oct 2011 at 7:14

**Comment by GoogleCodeExporter** [comment]
```
I encounter the same problem recently after I update my purecodecs. It works 
before, but now broke.
```

Original comment by `fung...@gmail.com` on 1 Dec 2011 at 12:44

**Comment by GoogleCodeExporter** [comment]
```
Some problems with sync of audio and waveform have been fixed in SE 3.3.11

For codec problems with directshow do install LAV filters (or switch to VLC as 
video player in settings)
```

Original comment by `nikse.dk@gmail.com` on 6 Dec 2013 at 10:19
- Changed state: **Fixed**
